src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+$cell-day-size: 100px;
+$month-block-width: ($cell-day-size * 7) + 7;
+$days: mon 0,  tue 101,  wed 202,  thu 303,  fri 404,  sat 505,  sun 606;
```

Comment:
it is better to calculate the values for margin dynamically based on cell width and gap rather then hardcode.